spec/factories/users.rb

```diff
@@ -2,9 +2,9 @@
 
 FactoryBot.define do
   factory :user do
-    first_name { Faker::Name.first_name }
-    last_name { Faker::Name.last_name }
-    username { Faker::Internet.unique.username }
+    first_name { Faker::Name.first_name[0..49] }
```

Comment:
I'm surprised we've got names exceeding the 50 character limit here on any sort of regular basis.